docs/events.md

```diff
@@ -4,6 +4,20 @@ Events that TrenchBoot appeared on.
 
 ## Worldwide conferences
 
+* Xen Project Summit 2024
+    * [Slides](https://static.sched.com/hosted_files/xenprojectsummit2024/59/trenchboot_in_xen.pdf)
+* FOSDEM 2024
+    * [Video](https://video.fosdem.org/2024/aw1126/fosdem-2024-3724-trenchboot-project-status-update.av1.webm)
```

Comment:
@philipandag, it doesn't seem to be consistent with the rest of the document. Please note other conferences are linked as follows:

```
* <conference_name> <year>
    * Video: [<talk_title>](<talk_video_link>)
    * [Slides](<slides_link>)
```

Reply:
Thank you

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/compare/b81d13e816bf7fd747c61bb0eea65bbdc80c102d..e77edbe0711d15428aad1ef9d300428d97b86887#diff-e070a13a5415feaf83554427f172fb145dcb7c57e396319bcc967c496a3a43f8